docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+version: "3.8"
+networks:
+  intergration-test:
+    driver: bridge
+
+services:
+  db:
+    image: postgis/postgis
+    container_name: postgres
+    restart: on-failure
+    healthcheck:
+      test: /usr/bin/pg_isready -U postgres
+      interval: 5s
+      timeout: 10s
+      retries: 120
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: postgres
+    volumes:
+    - pg_data:${PGDATA:-/var/lib/postgresql/data}:rw
+    - ./docker/01-init-user.sh:/docker-entrypoint-initdb.d/01-init-user.sh
+    ports:
+      - 5432
+    networks:
+      - intergration-test
```

Comment:
I would rename the network to something more meaningful. Maybe "mapstore" ?

Reply:
fixed here https://github.com/geosolutions-it/MapStore2/pull/6490/commits/f69942616b6d08cba28293e1cc20cda263a5dc22